README
wt_cart is a small but powerful extension which "solely" adds a shopping cart
to your TYPO3 installation.
The extension neither allows you to create products, articles or categories
nor includes a checkout functionality. The extension depends on powermail
which helps you to integrate the checkout process. Furthermore powermail is
utilized for sending the order confirmation.

Homepage: http://www.wt-cart.com

ChangeLog
3.0.0
	wt_cart Development Team <info@wt-cart.com>

	* Release of wt_cart 3.0.0
	* [FEATURE] #62412 . Add template for confirmation page
	* [FEATURE] #62411 - Make wt_cart compatible with powermail 2.1.x
	* [FEATURE] #62300 - 6.2 compatibility
	* [BUGFIX] #62410 - Deletion of products in cart does not work
	* [BUGFIX] #62240 - Follow up on pid conf
	* [BUGFIX] #51237 - No quantity given
	* [BUGFIX] #42764 - Calculation of shipping cost in value ranges and free_from
	* [TASK] - Update manual

2.1.0
	wt_cart Development Team <info@wt-cart.com>

  * Release of wt_cart 2.1.0
  * [FEATURE] #62241 - Add new hooks/ signal slots
  * [TASK] #52049 - wt_cart with powermail 2.0.0 and caching issue
  * [BUGFIX] #51925 - Fatal error after submit - wrong $conf call
  * [BUGFIX] #51230 - Cannot redeclare class Variant
  * [FEATURE] #46668 - Support of multidimensional variants
  * [TASK] code cleanup (PHP)
  * [TASK] #62251 - Move documentation to ReST
  * [FEATURE] #11522 - Include payment gateway
  * [TASK] - Add new icon
  * [TASK] - Update manual

2.0.0
	wt_cart Development Team <info@wt-cart.com>

  * Release of wt_cart 2.0.0
  * [FEATURE] #46671 - Revision of tax system
  * [FEATURE] #46670 - wt_cart should be based on net prices (optionally)
  * [FEATURE] #46669 - Support for AJAX / eID
  * [FEATURE] #46668 - Support of multidimensional variants
  * [FEATURE] #41587 - Do not cache pi3 (mini cart)
  * [BUGFIX] #39093 - Request for not existing table field "l10n_parent"
  * [FEATURE] #38938 - Make wt_cart compatible with powermail 2.x
  * [TASK] - Complete rewrite of code (PHP)
  * [TASK] #37764 - Move marker documentation from HTML templates to manual
  * [TASK] - Update manual

1.4.2
	wt_cart Development Team <info@wt-cart.com>

	* [FEATURE] #33742 - Add min and max order quantity
	* [FEATURE] #32135 - SQL query becomes cObj
	* [TASK] - Code cleanup (PHP)
	* [BUGFIX] - Use language string 'wtcart_ll_special_each' for special service costs
	* [BUGFIX] - Fixed problem when adding tt_content elements to wt_cart via pi2

1.4.1
	wt_cart Development Team <info@wt-cart.com>

	* [FEATURE] #32244 - Add new constant to control position of currency symbol
	* [TASK] - Fixed some problems in HTML template (missing subpart in table based template and static order number string)
	* [BUGFIX] - Cleaned TypoScript Templates

1.4.0
	wt_cart Development Team <info@wt-cart.com>

	* Release of wt_cart 1.4.0
	* [FEATURE] #32243 - Implement mini cart
	* [FEATURE] #31693 - Allow more than one wt_cart plugin per domain record
	* [FEATURE] #31692 - Include order numbers
	* [FEATURE] #31691 - Advanced shipping and payment calculation
	* [BUGFIX] #27456 - Problems when adding more than one product
	* [BUGFIX] #10101 - Problem with localization of product title
	* [TASK] - Update manual
	* [INFO] Dirk Wildt (Netzmacher) leaves the development team

1.3.3
	2011-09-11
	Bjoern Jacob <bjoern.jacob@tritum.de>

    * [BUGFIX] - Removed some TS which sets standard payment and shipping
    * [BUGFIX] - Removed deprecated methods GPvar() and GParrayMerged()
    * [BUGFIX] - Clear powermail session data

1.3.2
	2011-05-25
	Bjoern Jacob <bjoern.jacob@tritum.de>

	* Release of wt_cart 1.3.2
	* [TASK] - Update manual

	Daniel Lorenz <info@capsicum-ug.de>

	* [FEATURE] #26813 - free shipping
	* [TASK] - Code cleanup (PHP and TS)
	
1.3.1
	2011-05-23
	Bjoern Jacob <bjoern.jacob@tritum.de>

	* Release of wt_cart 1.3.1

	Daniel Lorenz <info@capsicum-ug.de>

	* [BUGFIX] #25883 - Amount can't update
	* [BUGFIX] #9870 - Behaviour when adding product again to cart

1.3.0
	2011-05-25
	Bjoern Jacob <bjoern.jacob@tritum.de>

	* Release of wt_cart 1.3.0
	* [TASK] - Update manual

    2011-01-16
    Dirk Wildt <http://wildt.at.die-netzmacher.de>

    * [TASK] Update of manual
    * [FEATURE] #12283 - Extend wt_cart for related tables
    * [FEATURE] #12282 - Use wt_cart with tax rates instead of ids for tax rates
    * [FEATURE] #12278 - Variants
    * [FEATURE] #12254 - Prompt for error and query in case of an SQL error
	* [INFO] - Dirk Wildt (Netzmacher) joines the development team

1.2.1
    2010-12-21
    Bjoern Jacob <bjoern.jacob@tritum.de>

    * Release of wt_cart 1.2.1
    * [FEATURE] #11619 - Add table based template

1.2.0
    2010-12-16
    Bjoern Jacob <bjoern.jacob@tritum.de>

    * Release of wt_cart 1.2
    * [TASK] - Translation for french
    * [TASK] - Update manual
    * [INFO] - Daniel Lorenz (Capsicumnet) joines the development team
    * [TASK] - Changed user_wtcart_numberFormat() for better integration with quick_shop
    * [FEATURE] #11516 - Add calculation of costs for shipping and payment
    * [FEATURE] #11515 - New constants
    * [FEATURE] #11514 - Don't include CSS via TS, use static CSS file

2010-09-22
    Bjoern Jacob <bjoern.jacob@tritum.de>

    * [BUGFIX] - Replaced deleted HTML template
    * [TASK] - Removed tt_news examples and HTML template file
    * [TASK] #9869 - Update manual with example for displaying additional product data
    * [FEATURE] #9868 - Add new constants for taxes

2010-09-16
    Bjoern Jacob <bjoern.jacob@tritum.de>

    * [TASK] - Update manual
    * [FEATURE] #9134 - Price and Title from POST param
    * [FEATURE] #9146 - Message if you forgot to add the static template
    * [FEATURE] #9431 - wt_cart depents on TYPO3 4.3 or higher
    * [BUGFIX] #9150 - Changing GPvar in TypoScript (deprecated)

2010-06-29
    Bjoern Jacob <bjoern.jacob@tritum.de>
     
    * [ADD] - New constant for cart pid which can be used in your own order form
    * [ADD] - New constant for currency which allows other currencies than Euro
    * [TASK] - Update manual

	Bjoern Jacob <bjoern.jacob@tritum.de>

    * [TASK] #8237 and #7903 - Update manual
    * [BUGFIX] #7895 - MailID not recognized
    * [TASK] - Extension now beta

	Bjoern Jacob <bjoern.jacob@tritum.de>

    * [BUGFIX] - Fixed some problems in manual

	Bjoern Jacob <bjoern.jacob@tritum.de>


ext_emconf.php
<?php

/***************************************************************
 * Extension Manager/Repository config file for ext "wt_cart".
 *
 * Auto generated 29-10-2014 22:27
 *
 * Manual updates:
 * Only the data in the array - everything else is removed by next
 * writing. "version" and "dependencies" must not be touched!
 ***************************************************************/

$EM_CONF[$_EXTKEY] = array (
	'title' => 'Shopping Cart for TYPO3',
	'description' => 'Adds shopping cart(s) to your TYPO3 installation and utilizes powermail for checkout',
	'category' => 'plugin',
	'shy' => false,
	'version' => '3.0.0',
	'dependencies' => '',
	'conflicts' => '',
	'priority' => '',
	'loadOrder' => '',
	'module' => '',
	'state' => 'beta',
	'uploadfolder' => false,
	'createDirs' => '',
	'modify_tables' => '',
	'clearcacheonload' => true,
	'lockType' => '',
	'author' => 'wt_cart Development Team',
	'author_email' => 'info@wt-cart.com',
	'author_company' => '',
	'CGLcompliance' => NULL,
	'CGLcompliance_note' => NULL,
	'constraints' => 
	array (
		'depends' => 
		array (
			'php' => '5.1.0-0.0.0',
			'typo3' => '4.5.0-6.2.99',
			'powermail' => '',
		),
		'conflicts' => 
		array (
		),
		'suggests' => 
		array (
		),
	),
);

?>